crates/turborepo-auth/src/lib.rs

```diff
@@ -111,6 +111,33 @@ impl Token {
         Ok(true)
     }
 
+    async fn handle_sso_token_error<T: Client + TokenClient + CacheClient>(
```

Comment:
We can drop the unused traits from the generic to make it easier to test.
```suggestion
    async fn handle_sso_token_error<T: TokenClient>(
```